src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceSpecifications.kt

```diff
@@ -52,6 +52,7 @@ fun hasNomsIdIn(nomsIds: List<String>?): Specification<Licence>? = nomsIds?.let
   Licence::nomsId.`in`(nomsIds)
 }
 
+// TODO: This does not work - how does JPA specification deeply get Licence::responsibleCom.staffIdentifier? Its not currently used
 fun hasStaffIdIn(staffIds: List<Int>?): Specification<Licence>? = staffIds?.let {
-  Licence::comStaffId.`in`(staffIds)
+  Licence::responsibleCom.`in`(staffIds)
```

Comment:
Need to come back to this.. any ideas?

Reply:
I'll take a look when doing next PR to add in probation area / Pdu / ldu to licence when its created.

Reply:
Yes, have some ideas on how to solve this.